pkg/drivers/dqlite/no_dqlite.go

```diff
@@ -4,12 +4,12 @@ package dqlite
 
 import (
 	"context"
-	"fmt"
+	"errors"
 
 	"github.com/rancher/kine/pkg/drivers/generic"
 	"github.com/rancher/kine/pkg/server"
 )
 
 func New(ctx context.Context, datasourceName string, connPoolConfig generic.ConnectionPoolConfig) (server.Backend, error) {
-	return nil, fmt.Errorf("dqlite is not support, compile with \"-tags dqlite\"")
+	return nil, errors.New("this binary is built without dqlite support, compile with \"-tags dqlite\"")
```

Comment:
```suggestion
	return nil, errors.New(`this binary is built without dqlite support, compile with "-tags dqlite"`)
```

Reply:
The convention for this codebase seems to be to use double quotes instead of backticks for unformatted and/or wrapped error messages - should I change the rest of them as well?

Reply:
To follow go standards, yes, however the conventions of the code base should take precedence.  I would say it's worthwhile to do it in a separate PR and ignore this change request comment for now.